### ISSUE TITLE
[Mekanism] Unable to extract Gas from PRC

### ISSUE THREAD

**Issue by Tyniann** [whatthedrunk/allthemods#116]
What happens: Built a setup to produce Ethylene, but the pressurized reaction chamber won't allow me to extract the gas, no matter how you configure the gas-configuration tab. Tested it for multiple hours in (creative)single and (survival)multiplayer. No matter what I tried, I couldn't get the Ethylene out of the machine - I used advanced pressurized tubes.
I post it here because I think it's modpack related because my online searching didn't yield any results regarding this issue - Either it seems to be pretty much working perfectly fine for everyone else or I am overlooking something very obvious.

What you expected to happen: Configure the gas-output of the machine, connect a pressurized tube to it and watch it filling with pinkish gas.

Steps to reproduce:

1. Build a simple setup to produce Ethylene (Hydrogen from the left, water from the back, bio mass and energy from the top).
2. Try to get the ethylene in a pressurized tube
3.
...

Version of All the Mods: 1.10.2 v1.12

Added mods (Optifine, Fastcraft, etc.):

1.
2.
3.
...

**Comment by Carrots4U** [comment]
Not gonna lie, this went a bit over my head, gonna ask @whatthedrunk's help on this one.

**Comment by Tyniann** [comment]
It's working now... *sigh*
I did a fresh installation of my minecraft files because we updated the server yesterday and after that I logged in and it just... worked. Maybe the mekanism config file was not uploaded correctly to my root server and got damaged? I don't really have a better explanation. Anyway - problem solved I guess. I will continue testing for a bit and if something comes up again I'll post it.
